packages/components/e2e/component-objects/src/main/java/org/talend/component/list/table/Table.java

```diff
@@ -0,0 +1,117 @@
+package org.talend.component.list.table;
+
+import org.talend.component.Component;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NotFoundException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+/**
+ * A List is used to easy access to WebElements of the react-talend-component List component - Table.
+ */
+public class Table extends Component {
+
+    private static final String NAME = "Table";
+
+    private static final String TABLE_SELECTOR = ".tc-list-table%s";
+
+    private static final String TABLE_COLUMN_HEADER_SELECTOR = ".ReactVirtualized__Table__headerColumn";
+
+    private static final String TABLE_COLUMN_HEADER_KEY_CLASS = TABLE_COLUMN_HEADER_SELECTOR + ".tc-list-cell-%s";
+
+    private static final String TABLE_ITEM_SELECTOR = ".ReactVirtualized__Table__row";
+
+    /**
+     * Constructor.
+     *
+     * @param driver Selenium WebDriver
+     */
+    public Table(final WebDriver driver) {
+        this(driver, null);
+    }
+
+    /**
+     * Constructor with table id
+     *
+     * @param driver Selenium WebDriver
+     */
+    public Table(final WebDriver driver, final String id) {
+        super(driver, NAME, String.format(TABLE_SELECTOR, id != null ? "#" + id : ""));
+    }
+
+    /**
+     * Get the headers.
+     *
+     * @return List of headers WebElement
+     */
+    public List<WebElement> getHeaders() {
+        return this.getElement().findElements(By.cssSelector(TABLE_COLUMN_HEADER_SELECTOR));
+    }
+
+    /**
+     * Get a specific header, identified by the column key.
+     * You can get this key from the className on header element : tc-list-cell-(columnKey).
+     *
+     * @param columnKey The column key
+     * @return The header WebElement
+     */
+    public WebElement getHeader(final String columnKey) {
+        return this.getElement().findElement(By.cssSelector(String.format(TABLE_COLUMN_HEADER_KEY_CLASS, columnKey)));
+    }
+
+    /**
+     * Get all rendered items, represented by a row in the table.
+     *
+     * @return The list of Items
+     */
+    public List<Item> getItems() {
+        return this.getElement() //
+                .findElements(By.cssSelector(TABLE_ITEM_SELECTOR)) //
+                .stream() //
+                .map(webElement -> new Item(driver, webElement)) //
```

Comment:
If I don't put them, and format it with Talend config, it is less readable.
That's why I choose to put them.